section2/exercises/comparisons.js

```diff
@@ -19,15 +19,15 @@ var stringTeachers = "4";
 var numberDogs = 0;
 
 // EXAMPLE: log the result of the comparison: is numberTeachers greater than numberStudents?
-console.log("Is numberTeachers greater than numberStudents?", numberTeachers > numberStudents);
+console.log("is numberTeachers > numberStudents?", numberTeachers > numberStudents);
 // this should log: "Is numberTeachers greater than numberStudents?" false
 
 // YOU DO: log the result of the comparison: is numberTeachers less than numberStudents?
 // this should log: true
-
+console.log("is numberTeachers < numberStudents?", numberTeachers < numberStudents);
```

Comment:
I see. So I can remove the quotations and strings so that the terminal just prints whether it is true or false?
ie:

console.log(numberTeachers < numberStudents);

Reply:
This was in reference to the other exercises, sorry that wasn't super clear from where I left my comment! For the other similar exercises in `comparisons.js`, you only had the first argument (printing the descriptive string), but you didn't `console.log` the boolean result!